cmd/provenanced/config/toml.go

```diff
@@ -51,6 +51,7 @@ func WriteConfigToFile(configFilePath string, config *ClientConfig) {
 		panic(err)
 	}
 
+	//nolint:gosec
```

Comment:
Always include a comment on the nolint directives.

```suggestion
	//nolint:gosec // The config file should be readable by anyone.
```